.travis.yml

```diff
@@ -10,11 +12,12 @@ install:
 
 matrix:
     include:
-        - php: 5.3.3
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
         - php: 5.6
-        - php: 7.0
-        - php: hhvm
+          env: DEPENDENCIES='dev' SYMFONY_VERSION="2.8.*@dev"
+        - php: 5.6
+          env: DEPENDENCIES='dev' SYMFONY_VERSION="3.0.*@dev"
     fast_finish: true
+
+before_script:
+    - if [ "$DEPENDENCIES" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi;
```

Comment:
I don't think this is necessary when you use the stability flag with the Symfony version.